src/library/scala/math/Equiv.scala

```diff
@@ -38,6 +38,7 @@ trait Equiv[T] extends Any with Serializable {
 trait LowPriorityEquiv {
   self: Equiv.type =>
 
+  @deprecated("An implicit universal Equiv instance allows code to compile when it is comparing instances of types for which equality isn't well-defined or implemented (for example, comparing two Function1 instances doesn't generally provide a meaningful result). Use Equiv.universal explicitly instead. If you really want an implicit univeral Equiv intance despite the potential problems, you can write a method `implicit def universalEquiv[T]: Equiv[T] = universal[T]`", "2.12.7")
```

Comment:
This is a pretty long line. I briefly looked for other examples of long deprecation messages in the code base but didn't see much. Happy to reformat if desired.

Reply:
you can look at [`Ordering.DeprecatedFloatOrdering`](https://github.com/scala/scala/blob/57c228631215378eef9a983456ff84cedb63e4bf/src/library/scala/math/Ordering.scala#L370-L373) for an example of a long deprecation message.

Reply:
I'd suggesting explaining at any length you like in the Scaladoc, then putting something shorter in the deprecation message that references the Scaladoc.

Reply:
@SethTisue I attempted to follow your advice. Let me know what you think.